src/Enterspeed.Source.UmbracoCms.V8/_content/App_Plugins/Enterspeed.Dashboard/Dashboard.Sub.Views/failedjobs.view.html

```diff
@@ -1,8 +1,13 @@
 ﻿<div ng-controller="FailedJobsController as vm" class="failedjobs-dashboard">
 
-    <umb-load-indicator ng-if="vm.loadingFailedJobs">
+    <umb-load-indicator ng-if="vm.loadingFailedJobs || vm.loadingConfiguration">
     </umb-load-indicator>
 
+    <div ng-if="!vm.runJobsOnServer" class="info-box">
+        <strong>Enterspeed jobs will not run on this current server.</strong><br />
+        Enterspeed jobs will only run on servers with Umbraco role of <i>Master</i> or <i>Single</i>. Current Umbraco server role is <i>{{vm.serverRole}}</i>.
+    </div>
```

Comment:
Should we add something about "Enterspeed.RunJobsOnAllServerRoles" as an option?